apps/mobile-wallet/src/components/layout/BottomModal.tsx

```diff
@@ -267,11 +282,25 @@ const Handle = styled(Animated.View)`
   margin-top: -15px;
 `
 
+const Title = styled(AppText)`
+  flex: 1;
+  text-align: center;
+`
+
 const Navigation = styled(Animated.View)`
-  align-items: flex-end;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 0 ${DEFAULT_MARGIN - 1}px;
+`
+
+const NavigationButtonContainer = styled.View<{ align: 'right' | 'left' }>`
+  width: 10%;
```

Comment:
That's one way to do it! But when the width of the close button is larger than 10% of the container, things would be misaligned.. This can happen on small screens.

<img width="295" alt="image" src="https://github.com/user-attachments/assets/cd417303-9850-46a2-9380-284338ec5a9b">


For perfect alignment I can think of 2 options:
1. render an invisible button to ensure both left and right flexbox items have exactly the same width 👍 
2. measure the width of the last flexbox item and programmatically set the width of the first item 👎

Reply:
I'm not sure I'm getting that! What I want is the left and right margin of the header to stay consistent. What would be misaligned? We can have a tiny call to clarify this :)

Reply:
I gave it another thought. It's fine with 10% both sides. Resolving.